linera-execution/src/pricing.rs

```diff
@@ -33,10 +56,25 @@ impl Pricing {
         Ok(self.messages.try_mul(size)?)
     }
 
-    pub fn storage_price(&self, data: &impl Serialize) -> Result<Amount, PricingError> {
+    pub fn storage_num_reads_price(&self, data: &impl Serialize) -> Result<Amount, PricingError> {
+        let size =
+            u128::try_from(bcs::serialized_size(data)?).map_err(|_| ArithmeticError::Overflow)?;
+        Ok(self.storage_num_reads.try_mul(size)?)
```

Comment:
`num_reads` suggests that this is about the fixed price per read operation, not about the price per byte.

Reply:
And yes, it is that. It is price per read operation, not price per byte.

Reply:
But you're still multiplying it with the size in bytes here.

Reply:
Ok, corrected.